### DISCUSSION TITLE
Organization matching query does not exist.

### DISCUSSION THREAD

**Discussion by iwanVgeelen** [minvws/nl-kat-coordination#52] [Q&A]
Ik heb inmiddels een account aangenmaakt. Alleen als ik organisation setup wil doen dan krijg ik: Organization matching query does not exist.
Wat kan ik hieraan doen?

**Comment by iwanVgeelen** [comment]
Met make kat?

En hoe is het account aangemaakt?

Graag iets meer context het lijkt alsof er handmatig in de django admin een account is aangemaakt? In dat geval moet er ook een "organization member" worden aangemaakt in de admin die dat account aan de dev org koppelt

**Comment by Mergpijp** [comment]
Met make kat ja. Ik zal het proberen maar denk dat het het niet verhelpt.

**Comment by underdarknl** [comment]
@dekkers could you have a look / help out?

**Comment by Mergpijp** [comment]
Hey, ik ben eruit moest een organization en organization member aanmaken en verder een groep genaamd 'redteam' dank voor de hulp.

**Comment by dekkers** [comment]
Even verplaatst naar discussions omdat dat een betere plek is dan github issues. Die organization en member zou aangemaakt moeten worden door "setup_dev_account" commando wat uitgevoerd zou moeten worden als je "make kat" draait. Waarschijnlijk kan je niet meer zien wat er tijdens "make kat" fout ging dat dit niet aangemaakt is?
